supplementary/alica_ros2/alica_ros2_turtlesim/src/base_node.cpp

```diff
@@ -43,7 +43,7 @@ int main(int argc, char** argv)
         RCLCPP_ERROR(nh->get_logger(), "Master plan or roleset location is not available");
         return 0;
     }
-
+    std::this_thread::sleep_for(1s);
```

Comment:
Is there a reason that we need this additional sleep? Maybe we can add a comment on why this is needed.

Reply:
@bjoernschroeder Dhananjay's internship is over so Likely you or Luca will be taking this up.  Just consider these PRs as a start.  They also cant run automated at the moment unless we can run alica turtlesim without the UI